Graphics/Vulkan/VulkanGraphicsDevice.cpp

```diff
@@ -116,16 +124,16 @@ namespace NovelRT::Graphics::Vulkan
             finalOptionalExtensions.emplace_back(requestedOptionalExt);
         }
 
-        std::vector<std::string> allExtensions = EngineConfig::RequiredVulkanPhysicalDeviceExtensions();
+        // TODO: EngineConfig was here
+        std::vector<std::string> allExtensions{requiredDeviceExtensions};
```

Comment:
We'll need to come back for this one.

Reply:
I'm not sure this is necessary anymore now that the extensions are passed in CreateLogicalDevice